chore: enable windows binary build in CI

For some random reason the windows binary build doesn't work on GitHub Actions:

```
--> windows/amd64 error: exit status 2
Stderr: # github.com/googleapis/gapic-showcase/cmd/gapic-showcase
cmd/gapic-showcase/run.go:41:80: undefined: syscall.SIGSTOP
```

Need to figure out how to fix that and reenable the build for the release asset.

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -89,11 +89,8 @@ jobs:
             arch: arm
           - os: darwin
             arch: amd64
-          # Disable windows build because it is not compiling for some reason.
-          # TODO(noahdietz): https://github.com/googleapis/gapic-showcase/issues/603
-          #
-          # - os: windows
-          #   arch: amd64
+          - os: windows
+            arch: amd64
     steps:
       - uses: actions/setup-go@v2
         with:

--- a/cmd/gapic-showcase/run.go
+++ b/cmd/gapic-showcase/run.go
@@ -38,7 +38,7 @@ func init() {
 			cmuxServer := CreateAllEndpoints(config)
 
 			done := make(chan os.Signal, 2)
-			signal.Notify(done, os.Interrupt, os.Kill, syscall.SIGTERM, syscall.SIGHUP, syscall.SIGSTOP)
+			signal.Notify(done, os.Interrupt, os.Kill, syscall.SIGTERM, syscall.SIGHUP)
 			go func() {
 				sig := <-done
 				stdLog.Printf("Got signal %q", sig)

